WalletWasabi/WabiSabi/Client/SmartRequestNode.cs

```diff
@@ -10,6 +10,10 @@ namespace WalletWasabi.WabiSabi.Client;
 
 public class SmartRequestNode
 {
+	// Limit reissuance requests at the same time when coinjoining with multiple wallets to avoid overloading tor.
+	private static readonly int MaxInputsRegistrableByWallet = 10;
```

Comment:
Why `MaxInputsRegistrableByWallet`? 

AFAIK this is output registration.
How about `MaxReissuanceRequestParallel`? or something like this?